packages/flutter/test/rendering/table_test.dart

```diff
@@ -121,52 +132,225 @@ void main() {
         ' │   size: Size(100.0, 10.0)\n'
         ' │   additionalConstraints: BoxConstraints(w=30.0, h=10.0)\n'
         ' │\n'
-        ' ├─child (3, 0) is null\n'
```

Comment:
No longer allow the existence of null RO for `_children`

Reply:
isn't that a problem?

Reply:
Refer to https://github.com/flutter/flutter/pull/69670#discussion_r543605195